pkg/cloud-init/cloud-init.go

```diff
@@ -563,8 +570,10 @@ func GenerateLocalData(vmiName string, namespace string, data *CloudInitData) er
 			data.ConfigDriveMetaData = &ConfigDriveMetadata{
 				InstanceID: fmt.Sprintf("%s.%s", vmiName, namespace),
 			}
+			if instanceType != "" {
+				data.ConfigDriveMetaData.InstanceType = instanceType
+			}
```

Comment:
same here, if instance type isn't a pointer, then both of these are equivalent i think.

`
			if instanceType != "" {
				data.ConfigDriveMetaData.InstanceType = instanceType
			}
`
and 

`
				data.ConfigDriveMetaData.InstanceType = instanceType
`